task3/simplyTable.js

```diff
@@ -0,0 +1,18 @@
+let mybtn = document.getElementById('btn')
+
+mybtn.onclick = function () {
+    setTimeout(function () {
+        const myStatus = document.querySelectorAll('.pending')
+        let i = 0
+        let interval = setInterval(function () {
+            console.log('Check started')
+            if (myStatus[i].innerHTML === 'Pending') {
+                myStatus[i].innerHTML = 'Done'
+                myStatus[i].style.color = '#85929E'
+            }
+            if (++i === myStatus.length) {
```

Comment:
It's not a good idea to use such iterators. Try to avoid it in the future due to high risks.